ros:
  bag: "/data/euroc/rosbags/imu_april.bag"
  cam0_topic: "/cam0/image_raw"
  cam1_topic: "/cam1/image_raw"

settings:
  data_path: "/tmp/cam_april"
  results_fpath: "/tmp/cam_april/calib_results.yaml"
  imshow: true

calib_target:
  target_type: 'aprilgrid'  # Target type
  tag_rows: 6               # Number of rows
  tag_cols: 6               # Number of cols
  tag_size: 0.088           # Size of apriltag, edge to edge [m]
  tag_spacing: 0.3          # Ratio of space between tags to tagSize
                            # Example: tagSize=2m, spacing=0.5m --> tagSpacing=0.25[-]

cam0:
  resolution: [752, 480]
  proj_model: "pinhole"
  dist_model: "radtan4"

cam1:
  resolution: [752, 480]
  proj_model: "pinhole"
  dist_model: "radtan4"
